app/assets/javascripts/tagging.js

```diff
@@ -2,6 +2,14 @@ function addTag(tagname, selector) {
 
   selector = selector || '#tagform';
 
+  if (tagname.indexOf("place") != -1) {
+    place = tagname.split(":")[1];
+    place.replace("-", " ");
+    geocodeStringAndPan(place);
+    var choice = prompt("This looks like a location: A) Geocode B) Leave");
+    while(choice == NULL)
+  }
```

Comment:
I think the `while` may not be needed? the `prompt()` should hang up the process until it's submitted.

Reply:
Awesome